.github/workflows/check-migrations.yml

```diff
@@ -0,0 +1,23 @@
+name: Check Migrations
+
+on:
+  pull_request:
+    paths:
+      - 'etna/*/migrations/**'
+
+jobs:
+  check-migrations:
+    name: Check Migrations
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
```

Comment:
Could the version number work with a generic variable used elsewhere maybe `.github/workflows/_tests.yml` ?

Reply:
Good shout Allan, thank you 😄 I'd forgotten we had that

Reply:
Ok - we can't do this for the `pull_request` type of call, only `workflow_dispatch`, so I will have to leave this hard-coded. 😔